### DISCUSSION TITLE
Trendeki kuş sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#299] [Zeka Sorusu]
120m uzunluğunda ve 60km/saat hızla hareket eden trenin en arkasından sabit hızla trenle aynı yönde hareket eden bir kuşun trenin en önüne gidip hiç zaman kaybetmeden en sonuna geri dönmesi için gereken süre 21 saniyedir.
Kuşun hızı kaç km/saattir?

**Comment by slos8888** [comment]
kuşun hızı= (120 m / 21 saniye) x 3600 km/saniye

Kuşun hızı=20.57 km/saattir.

Sonuç olarak, kuşun hızı yaklaşık 20.57 km/saat olarak bulunur.
